portfolio/src/main/java/com/google/sps/servlets/AboutTranslateServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translateabout")
+public class AboutTranslateServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String par = request.getParameter("par");
```

Comment:
Please use more descriptive variable names, paragraph is fine, rename parameter names as well